### ISSUE TITLE
How to launch Jekyll Admin as a seperate service

### ISSUE THREAD

**Issue by csharpee** [jekyll/jekyll-admin#681]
### Description:
Am I able to launch Jekyll Admin as an independent service without launching Jekyll serve? Jekyll admin could easily compete with Strapi, although at the moment Strapi does have a Jekyll plug-in.

Having Jekyll-Admin locally isn’t really what a ‘developer’ needs. That type of thing is for editors and non-technical web admins.

**Comment by ashmaroli** [comment]
Hello @csharpee Jekyll Admin was not built to be used as a SAAS. In fact, it was designed for layman editors and non-technical folks.

I'm sorry, but your expectation is exactly opposite of what Jekyll Admin is..:
- We expect our users to invoke `jekyll serve` to initialize the API server which is then utilized to render the admin frontend.
- We expect the API (JSON) to be a reflection of the Jekyll build process. i.e. expose both unbuilt raw source contents *and* the built static content.
- We expect the API to be used only to render the admin frontend unlike being used to render the user's headless site itself.

**Comment by csharpee** [comment]
Hi @ashmaroli :wave:  

It seems I must create my own basic auth with nginx, create a simple back-end dashboard and point to api location for basic CRUD like operation using javascript.

I don't want to use react or other javascript pancake framework. All I need is Jekyll SSG and a couple other "plugins" like capacitor to pump out iOS, Android and PWA without all the complex javascript pancakes. 

All I need is HTML, CSS and JS to create for mobile and web.

I use my own GIT server with automation and also serves the static site like GitHub pages. 

In short, I want Jekyll to have a Jekyll authenticated dashboard  that can be used for individual and multiple sites.

On the development team, the Jekyll Admin plugin isn't used because devs edit the source code with a text editor like vs or atom. I want to hand-off the finalized website or app to a community or business without them having to be on the "dev" team. The clients should have a similar process of login and manage basic things like posts, pages, etc...

**Comment by derek-adair** [comment]
> I don't want to use react or other javascript pancake framework. All I need is Jekyll with Liquid super powers.

Me too buddy... me too.  However, its clear thats not what this project is for!  Therefor... WE FORK!

I'll be meditating on this and how it can be forked to provide the functionality you've described.  Currently i intend on setting up a series of dependent docker containers that will do this; but it will likely not have any use outside of my specific designs.  Reach out to me directly if you would like to collaborate on this.

**Comment by derek-adair** [comment]
Also for what its worth this project _is_ react/redux
